Doc/faq/library.rst

```diff
@@ -774,10 +774,10 @@ have to check what's returned on your system.
 You can use the ``connect_ex()`` method to avoid creating an exception.  It will
 just return the errno value.  To poll, you can call ``connect_ex()`` again later
 -- ``0`` or ``errno.EISCONN`` indicate that you're connected -- or you can pass this
-socket to select to check if it's writable.
+socket to ``select()`` to check if it's writable.
 
 .. note::
-   The :mod:`asyncore` module presents a framework-like approach to the problem
+   The :mod:`asyncio` module presents a framework-like approach to the problem
```

Comment:
Asyncore is a separate module from asyncio, see https://docs.python.org/3.5/library/asyncore.html

Reply:
Of course but it is now deprecated and all all its functionnality has been moved to asyncio, hence the proposed update.

Reply:
> Of course but it is now deprecated and all all its functionnality has been moved to asyncio, hence the proposed update.

My point was that the two are not exactly one-to-one. 

I agree that this section should be updated to recommend asyncio, but the surrounding text should be adjusted a bit as well. Their purposes are similar, but the existing text is more aimed at asyncore. Asyncio is a general purpose concurrency library, whereas asyncore was more of a framework for a specific purpose.

```
The :mod:`asyncio` module provides a general purpose concurrency
library, which can be used for writing non-blocking network code.
 ```

Reply:
OK great, I've committed your proposal.